### PR TITLE
feat: add program registry to cards

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: regen-network/rnd-dev-team#1653
Sanity PR: https://github.com/regen-network/regen-sanity/pull/46

Add program info to:
- Project Card
- Credit Class Card
- Featured Card
- Ecological Credit Card

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1. Home: https://deploy-preview-2015--regen-registry.netlify.app/
2. Project: https://deploy-preview-2015--regen-registry.netlify.app/project/C01-001
3. Buyers: https://deploy-preview-2015--regen-registry.netlify.app/buyers

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by flagrede** [comment]
@erikalogie you can test that the program has been added to cards, I've put the links in the description.

File: web-registry/src/lib/normalizers/projects/normalizeProjectsWithCreditClass.ts [hunk 1]
```diff
@@ -19,13 +23,15 @@ interface Params {
...
   projectPagesMetadata?: ProjectPageMetadataLD[];
+  projectParties?: Maybe<PartyFieldsFragment | undefined>[];
   classesMetadata?: (CreditClassMetadataLD | undefined)[];
   sanityCreditClassData?: AllCreditClassQuery;
 }
 
 export const normalizeProjectsWithCreditClass = ({
   projectsMetadata,
   projectPagesMetadata,
+  projectParties,
```
Comment (blushi) [inline web-registry/src/lib/normalizers/projects/normalizeProjectsWithCreditClass.ts:34]: should we use a more explicit name like programParties or registryParties?

**Review by blushi — COMMENTED** [review]
It seems like the program is not appearing on the projects cards on https://deploy-preview-2015--regen-registry.netlify.app/projects/1

File: web-registry/src/lib/normalizers/projects/normalizeProjectsWithCreditClass.ts (hunk 1 above)
Comment (flagrede) [reply web-registry/src/lib/normalizers/projects/normalizeProjectsWithCreditClass.ts:34]: `programParties` sounds good to me, will update it.

**Comment by flagrede** [comment]
@blushi I added the program on cards from the projects page, thanks for catching that.

File: web-components/src/components/cards/ProjectCard/ProjectCard.ImageChildren.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  const theme = useTheme();
+
+  return (
+    <>
+      {program && (
+        <Box
+          sx={{
+            position: 'absolute',
+            bottom: theme.spacing(3.75),
+            left: theme.spacing(5),
```
Comment (ryanchristo) [inline web-components/src/components/cards/ProjectCard/ProjectCard.ImageChildren.tsx:19]: Any reason for this not being 5,5 ? Seems slightly off to me but maybe this is as intended.

**Review by ryanchristo — APPROVED** [review]
tACK. Just one small nit but also may not be necessary.

File: web-components/src/components/cards/ProjectCard/ProjectCard.ImageChildren.tsx (hunk 2 above)
Comment (ryanchristo) [reply web-components/src/components/cards/ProjectCard/ProjectCard.ImageChildren.tsx:19]: cc @erikalogie

Comment (ryanchristo) [reply web-components/src/components/cards/ProjectCard/ProjectCard.ImageChildren.tsx:19]: I know we have a few new spacing issues so just want to double check before we merge.

Comment (ryanchristo) [reply web-components/src/components/cards/ProjectCard/ProjectCard.ImageChildren.tsx:19]: Screenshot from 2023-08-14 10-34-29

**Pull request merged**